decidim-core/lib/decidim/core.rb

```diff
@@ -607,7 +607,7 @@ def self.organization_settings(model)
   end
 
   # Defines the time after which the machine translation job should be enabled.
-  # In some cases, it is required to have a delay, otherwise the ttanslation job will be discarded:
+  # In some cases, it is required to have a delay, otherwise the translation job will be discarded:
```

Comment:
```suggestion
  # In some cases, like when sending emails with notifications that have automatic translations, it is required to have a delay, otherwise the translation job will be discarded:
```
Let's clarify in which cases

Reply:
@andreslucena , i have changed the documentation fragment with : 

```
  # Defines the time after which the machine translation job should be enabled.
  # In some cases, like when Workers is processing faster than ActiveRecord can commit to Database,
  # it is required to have a delay, to prevent any discarding with
  # Decidim::MachineTranslationResourceJob due to a ActiveJob::DeserializationError.
  # In some Decidim Installations, ActiveJob can be configured to discard jobs failing with
  # ActiveJob::DeserializationError
```